uk.co.compucorp.civicrm.hrleaveandabsences/hrleaveandabsences.php

```diff
@@ -369,9 +369,9 @@ function hrleaveandabsences_civicrm_tabset($tabsetName, &$tabs, $context) {
   if ($tabsetName == 'civicrm/contact/view') {
     $contactId = $context['contact_id'];
     $tabs[] = [
-      'id'        => 'absence',
+      'id'        => 'leave',
```

Comment:
Note that when the id of the tab changes, URLs that points to the absence tab on the contact summary page also needs to change e.g https://github.com/civicrm/civihr/blob/223dee88266bc0bf9d9c48f9cb1609c3e3040708/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/absence-tab/components/annual-entitlements.component.spec.js#L84

In this case the selectedChild=leave will be the correct url. Not sure we have many cases of that but will be a good idea to scan the code.